README.md

```diff
@@ -300,6 +306,74 @@ let result = try await openAI.images(query: query)
 
 ![Generated Image](https://user-images.githubusercontent.com/1411778/213134082-ba988a72-fca0-4213-8805-63e5f8324cab.png)
 
+### Create Image Edit
+
+Creates an edited or extended image given an original image and a prompt.
+
+**Request**
+
+```swift
+public struct ImageEditsQuery: Codable {
+    /// The image to edit. Must be a valid PNG file, less than 4MB, and square. If mask is not provided, image must have transparency, which will be used as the mask.
+    public let image: String
```

Comment:
Could you describe why the image has a `String` type? Looks like, this should be a multipart form data request where the image and mask should be uploaded.